routes/orders.js

```diff
@@ -0,0 +1,69 @@
+const ordersRouter = require("express").Router();
+const { getAllOrders, getCartByUser, createOrder, updateOrder } = require("../db")
+const { isLoggedIn, isAdmin } = require("./util")
+
+ordersRouter.get("/", isLoggedIn, isAdmin, async (req, res, next) => {
+    try {
+        const allOrders = await getAllOrders();
+        res.send(allOrders);
+    } catch (error) {
+        throw error;
+    };
+});
+
+ordersRouter.get("/cart", isLoggedIn, async (req, res, next) => {
+    const { id } = req.body
+    try {
+        const userCart = await getCartByUser({id});
+        res.send(userCart)
+    } catch(error) {
+        throw error;
+    };
+});
+
+ordersRouter.post("/", isLoggedIn, async (req, res, next) => {
+    const { status, userId } = req.body;
+    try {
+        const newOrder = await createOrder({status, userId})
+        res.send({
+            name: "OrderCreated",
+            message: "Your order has been made"
+        }, newOrder)
+    } catch(error) {
+        throw error;
+    };
+});
+
+ordersRouter.patch("/:orderId", isLoggedIn, isAdmin, async (req, res, next) => {
+    const { orderId } = req.params;
+    const { status, userId } = req.body;
+    try {
+        const updatedOrder = await updateOrder({ orderId, status, userId })
+        res.send({
+            name: "OrderUpdate",
+            message: "The order has been updated"
+        }, [updatedOrder]);
+    } catch (error) {
+        throw error;
+    };
+});
+
+ordersRouter.delete("/:orderId", isLoggedIn, isAdmin, async (req, res, next) => {
+    const { orderId } = req.params;
+    const { status, userId } = req.body;
+    try {
+        const deletedOrder = await deletedOrder({orderId, status, userId})
```

Comment:
I think we need to write this deleteOrder function and import it.

Reply:
Commented out for now as this component was written before deleteOrder has been created before another collaborator.